testing/btest/scripts/base/protocols/dns/dynamic-update.zeek

```diff
@@ -1,6 +1,12 @@
-# @TEST-DOC: Tests that a DNS dynamic update packet doesn't error but reports an unknown opcode weird
-# @TEST-EXEC: zeek -b -C -r $TRACES/dns/dynamic-update.pcap %INPUT
-# @TEST-EXEC: btest-diff weird.log
+# @TEST-DOC: Tests that a DNS dynamic update packet is processed.
+# @TEST-EXEC: zeek -b -C -r $TRACES/dns/dynamic-update.pcap %INPUT >out 2>&1
+# @TEST-EXEC: btest-diff out
+# @TEST-EXEC: ! test -f weird.log
 
 @load base/frameworks/notice/weird
 @load base/protocols/dns
+
+event dns_dynamic_update(c: connection, msg: dns_msg, zname: string, zclass: count)
+	{
+	print msg, zname, zclass, DNS::classes[zclass];
+	}
```

Comment:
I'm only skimming. The juicy info I believe is in the updates section in the the test pcap - `nwin2.stratolab.org` and 192.168.1.106 as IP.


I wonder if `dns.log` is even the right place to put this, or whether a dedicated `dns_update.log` might be better. Maybe adding `dns.log` to this test would give some more ground/example to discuss, but I could certainly see `dns.log` being just about "query" and going somewhere else for "update".

<img width="856" height="662" alt="image" src="https://github.com/user-attachments/assets/e7fb84e9-d19b-4061-a088-25e17c766da2" />

Reply:
If we go for a separate `dns_update.log`, I guess I could see starting with just a log that lists each of them
```
uid | transaction_id (top-level) | class | type | name | data | success
```

Or grouping by transaction_id as you indicate. I think this is log design and might be good to have a conversation with the team / users outside of the PR?